src/main/webapp/assets-lti/js/controllers/controllers.js

```diff
@@ -350,6 +364,9 @@ canvasSupportApp.controller('addUserController', ['Friend', '$scope', '$rootScop
     else {
       $scope.coursemodal.loadingLookupFriend = false;
       $scope.failedValidation = true;
+      $scope.$evalAsync(function() { 
+        focus('failedValEmail');
```

Comment:
My only comment is that failedValEmail could be failedToValidateEmail or something more explicit for future developers. Otherwise, looks good.